microsoft-edge/webview2/index.md

```diff
@@ -136,7 +138,7 @@ WebView2 apps can run on the following versions of Windows.
 *   Windows Server 2008 R2 \*\*
 
 > [!IMPORTANT]
-> \*\* WebView2 support for Windows 7 and Windows Server 2008 R2 has the same support cycle as Microsoft Edge.  For more information, navigate to [Microsoft Edge supported Operating Systems](/deployedge/microsoft-edge-supported-operating-systems).
+> WebView2 support for Windows 7 and Windows Server 2008 R2 will have the same support timeline as Microsoft Edge.  For more information, navigate to [Microsoft Edge supported Operating Systems](/deployedge/microsoft-edge-supported-operating-systems).
```

Comment:
Removing ** here might not work, because ** remains, two lines above, at "Windows Server 2008 R2 \*\*".

Reply:
Generally, usually, when possible, style guides for dev docs say to try to use present tense.  This spot could be an exception where future tense is required/appropriate.
https://docs.microsoft.com/en-us/style-guide/grammar/verbs#verb-tense
https://developers.google.com/style/timeless-documentation

Reply:
The word 'navigate' doesn't appear in MWSG.  The word 'see' is standard for "Click this provided link and read the article that immediately appears (no multi-step navigation/wayfinding/course-plotting is required)."
```suggestion
> WebView2 support for Windows 7 and Windows Server 2008 R2 will have the same support timeline as Microsoft Edge.  For more information, see [Microsoft Edge supported Operating Systems](/deployedge/microsoft-edge-supported-operating-systems).
```
https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/accessibility-terms